docs/languages/python.md

```diff
@@ -33,14 +33,13 @@ To experience Python, create a file (using the [File Explorer](/docs/getstarted/
 print("Hello World")
 ```
 
-The Python extension then provides shortcuts to run Python code in the currently selected interpreter (**Python: Select Interpreter** in the Command Palette):
+The Python extension then provides shortcuts to run Python code using the currently selected interpreter (**Python: Select Interpreter** in the Command Palette), just click the **Run Python File in Terminal** play button in the top-right side of the editor.
```

Comment:
maybe simplify and break this into two sentences:
"To run the active Python file, select the **Run Python File in...."